File: #100daysofcodewithGFG/Mirror Tree.cpp
class Solution {
  public:
    // Function to convert a binary tree into its mirror tree.
    void func(Node* node){
        if (node == NULL){
            return;
        }
        swap(node->left,node->right);
        func(node->left);
        func(node->right);
    }
    void mirror(Node* node) {
        // code here
        func(node);
    }
};

File: #100daysofcodewithGFG/Mirror Tree.py
class Solution:
    #Function to convert a binary tree into its mirror tree.
    def mirror(self,root):
        # Code here
        def func(root):
            if root == None:
                return
            temp = root.right
            root.right = root.left
            root.left = temp
            func(root.left)
            func(root.right)
        head = root
        func(root)
        return head
